datalad/resources/website/index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Datalad Repository</title>
+        <link rel="shortcut icon" type="image/x-icon" href="http://datalad.org/pics/favicon-16x16.png" />
+	<link rel="stylesheet" type="text/css" href="http://ajax.aspnetcdn.com/ajax/jquery.dataTables/1.10.5/css/jquery.dataTables.css">
+        <style type="text/css" class="init">
+          body{
+           max-width: 1200px;
+           margin: 2em auto;
+         }
+	 td.dir {font-weight: bold; color: #00A1CB; }
+	 td.git {font-weight: bold; color: #00BA88; }
+	 td.annex {font-weight: bold; color: #EC6C20; }
+	 td.link-broken {color:#E53B51; }
+	 td.link {color: #FFD03C;  }
+	 td.file {color: #67CDDC;  }
+	</style>
+    </head>
+    <body>
+        <h1><img src='http://datalad.org/pics/favicon.svg' width="24">Datalad Repository</h1>
+	<table id="directory" class="display" width="100%" cellspacing="0"></table>
+	<script type="text/javascript" charset="utf8" src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.12.3.min.js"></script>
+	<script type="text/javascript" charset="utf8" src="http://ajax.aspnetcdn.com/ajax/jquery.dataTables/1.10.5/jquery.dataTables.min.js"></script>
+	<script type="text/javascript" charset="utf8" src="https://raw.githubusercontent.com/blueimp/JavaScript-MD5/master/js/md5.min.js"></script>
```

Comment:
not sure what is the ideal setup would be.  I see possible scenarios, but in each of those we better have copies of those js/css under our `datalad/resources/web` as I have said before:
1. fully self sufficient "independent" -- all js/css are deployed for every dataset. *pros*: guaranteed to be consistent with index.html script deployed within that dataset, not dependent on github etc being available/online.  *cons*: for every dataset browser would need to fetch exactly the same (often) copy of them, thus slow
2. fully self sufficient "top repo only": we revert back to provide index.html only within top dataset and then revert back and do not have those "jumps" into subdatasets urls, so path to sub-datasets would always be within path option of url (e.g. http://datasets.datalad.org/webtest/?dir=ds000001/).  And then deploy all those .js /.css files for the super-dataset.  *pros*: all of 1. without cons of 1 since we have a single copy/url to get to them. Might also be easier to allow users for customizations *cons*: no "direct" urls to sub-datasets (only through path= option, rewrites on server side probably possible if necessary)
3. dependent on some delivery website (like githubusercontent):  so we have our own versioned copies (under `datalad/resources/web`) and point urls to them.  *pros*: might be faster access to them from github?  *cons* not guaranteed access to them, couldn't be used offline (e.g. from some quick local webserver offline)

So I think 2. is probably the way to go unless I see some showstopper. @hanke -- welcome to chime in.